portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
```

Comment:
This file doesn't use `mount.jpg`. I'm not sure of what the intended use of that image is, but if it's not going to be used, remove it so there's no unused assets in the project